tests/Unit.Tests/Controllers/SubjectGetTests.cs

```diff
@@ -54,29 +59,25 @@ public void SetUp()
                 new GoogleAnalyticsClient(null, null));
             var tempDataMock = new Mock<ITempDataDictionary>();
             _filterController.TempData = tempDataMock.Object;
-
         }
 
         [Test]
-        [Ignore("ancient test isn't working, PRs welcome")]
         public void GivenSomeSubjects_WhenSubjectGetCalledWithNullSubjectFilter_ThenViewModelHasAllSubjects()
```

Comment:
GivenSomeSubjects_WhenSubjectGetCalled`WithNullSubjectFilter`_ThenViewModelHasAllSubjects

Im not too sure the test name makes sense